### ISSUE TITLE
accessibility: keyboard event cannot emit click events

### ISSUE THREAD

**Issue by jmuk** [Polymer/paper-button#11]
Normal &lt;button> emits click event on keypress of enter key or space when it's focused, but &lt;paper-button> doesn't respond to keyboard events at all.

This could be a bit more serious considering accessibility.
